### PR TITLE
[nnc][quantization] quantized ops for BI bytedoc via aten

### PR DESCRIPTION
**Pull request by IvanKobzarev**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #68790
* #68736

Differential Revision: [D32609427](https://our.internmc.facebook.com/intern/diff/D32609427)

### CODE REVIEW

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 1]
```diff
@@ -249,6 +256,7 @@ void nnc_aten_quantized_conv2d_relu(
 }
 
 #ifndef _WIN32
+
```
Comment (ZolotukhinM) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:259]: Nit: unnecessary change

**Review by ZolotukhinM — APPROVED** [review]
👍

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32609427).
